### DISCUSSION TITLE
experiments page tagline grammar

### DISCUSSION THREAD

**Discussion by CatherineKlauss** [Qiskit/qiskit.org#2390] [Ideas]
currently is "Browse and contribute with innovatives ways of using quantum computing and Qiskit." which grammatically does not make sense. 

suggestion: "Browse and contribute to the fun and innovative applications of Qiskit."

**Comment by delapuente** [comment]
Per offline conversations, we don't want to include "fun" in the description. We think Qiskit community contributions can be sometimes fun but this is not the central point of the experiments section but, according to @y4izus, to drawe the attention of user into what the community has developed with Qiskit.

@CatherineKlauss is it possible to fix the grammar without altering the message?

**Comment by y4izus** [comment]
We are still having offline conversations about this issue. It has no sense to have this issue on the sprint backlog until we don't have a resolution. That's why I remove this issue from the sprint backlog
